web/src/hotkey.js

```diff
@@ -1189,6 +1190,9 @@ export function process_hotkey(e, hotkey) {
         case "compose_quote_message": // > : respond to selected message with quote
             compose_reply.quote_message({trigger: "hotkey"});
             return true;
+        case "compose_forward_message": // > : forward selected message
```

Comment:
The comment looks to be inaccurate about which key is being used.

Reply:
Fixed!